api/src/transactions.rs

```diff
@@ -1220,6 +1220,19 @@ impl TransactionsApi {
             _ => ExecutionStatus::MiscellaneousError(None),
         };
 
+        let stats_key = match txn.payload() {
+            TransactionPayload::Script(_) => "Script".to_string(),
+            TransactionPayload::ModuleBundle(_) => "ModuleBundle".to_string(),
+            TransactionPayload::EntryFunction(entry_function) => FunctionStats::function_to_key(
+                entry_function.module(),
+                &entry_function.function().into(),
+            ),
+            TransactionPayload::Multisig(_) => "MultiSig".to_string(),
```

Comment:
MultiSig has an entry function tied to it, so I would probably add that here as well.  It's in the inner type.